### DISCUSSION TITLE
Hi i need to know how to get the API key

### DISCUSSION THREAD

**Discussion by jerekali2005** [br-g/openf1#104] [General]
I am doing a school project and when using the API, I need the API key. I don't know if I have to create an account or something, since I read everything and I can't find anything. I would like to know if someone can help me. Thank you very much.

**Comment by br-g** [comment]
Hello,

There is no need for an API key!
Here is, for example, how you can fetch the list of GPs for the 2024 season: https://api.openf1.org/v1/meetings?year=2024
